Core/include/Acts/Geometry/DetectorVolume.hpp

```diff
@@ -328,6 +335,14 @@ class DetectorVolume : public std::enable_shared_from_this<DetectorVolume> {
   /// @return a boolean indicating if the objects are properly contained
   bool checkContainment(const GeometryContext& gctx, size_t nseg = 1) const;
 
+  /// build the bounding box
+  ///
+  void boundingBox(const GeometryContext& gctx);
+
+  /// Retrieve bounding box
+  ///
+  void setBoundingBox(const GeometryContext& gctx);
```

Comment:
I don't understand the difference between these two functions. One just calls the other? Can we get rid of one of them? Also I'd prefer if the name was more concrete, i.e. something like `createBoundingBox` or so.

Reply:
Ah, yeah no difference. I got rid of setBoundingBox and changed the name of boundingBox to createBoundingBox.